tools/ports/zlib.py

```diff
@@ -6,16 +6,16 @@
 import os
 import shutil
 
-TAG = 'version_1'
-HASH = '77f7d8f18fe11bb66a57e358325b7422d721f7b506bd63293cfde74079f958864db66ead5a36c311a76dd8c2b089b7659641a5522de650de0f9e6865782a60dd'
+TAG = 'v1.2.11'
+HASH = '104c62ed1228b5f1199bc037081861576900eb0697a226cafa62a35c4c890b5cb46622e399f9aad82ee5dfb475bae26ae75e2bd6da3d261361b1c8b996970faf'
```

Comment:
```suggestion
HASH = 'a42b8359e76cf7b3ae70bf31f0f8a8caa407ac80e8fe08b838076cd5e45ac2e685dae45eb59db2d25543fb3b5bd13b843a02bb8373cda704d7238be50d5e9c68'
```

Reply:
That's weird. I wonder why it's different.

$ wget https://github.com/madler/zlib/archive/v1.2.11.tar.gz
--2021-05-20 13:24:07--  https://github.com/madler/zlib/archive/v1.2.11.tar.gz
Resolving github.com (github.com)... 140.82.114.3
Connecting to github.com (github.com)|140.82.114.3|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://codeload.github.com/madler/zlib/tar.gz/v1.2.11 [following]
--2021-05-20 13:24:08--  https://codeload.github.com/madler/zlib/tar.gz/v1.2.11
Resolving codeload.github.com (codeload.github.com)... 140.82.112.10
Connecting to codeload.github.com (codeload.github.com)|140.82.112.10|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 644492 (629K) [application/x-gzip]
Saving to: ‘v1.2.11.tar.gz’

v1.2.11.tar.gz                100%[=================================================>] 629.39K  --.-KB/s    in 0.1s

2021-05-20 13:24:08 (4.29 MB/s) - ‘v1.2.11.tar.gz’ saved [644492/644492]

$ sha512sum v1.2.11.tar.gz
104c62ed1228b5f1199bc037081861576900eb0697a226cafa62a35c4c890b5cb46622e399f9aad82ee5dfb475bae26ae75e2bd6da3d261361b1c8b996970faf  v1.2.11.tar.gz